### PR TITLE
Fix link to contributing.md

### PR DESCRIPTION
**Pull request by emilbruckner**
CONTRIBUTING.md -> 404

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -713,7 +713,7 @@ Our Roadmap towards 2.0.0 [is public](https://github.com/zeit/next.js/wiki/Roadm
 
 ## Contributing
 
-Please see our [CONTRIBUTING.md](./CONTRIBUTING.md)
+Please see our [CONTRIBUTING.md](./contributing.md)
```
Comment (albinekb) [inline readme.md]: change the link text too?
`[contributing.md](./contributing.md)`

Comment (emilbruckner) [inline readme.md]: makes sense

**Review by timneutkens — APPROVED** [review]
Thanks! 🙌

File: readme.md (hunk 1 above)
Comment (timneutkens) [inline readme.md]: Fixed in #1190.

**Pull request merged**